packages/manager/apps/telecom/src/app/telecom/pack/move/translations/Messages_fr_FR.json

```
@@ -103,7 +103,8 @@
   "pack_move_modem_rental": "Location modem",
   "pack_move_installation_costs": "Frais d'installation",
   "pack_move_provider_infra_costs": "Frais supplémentaires",
-  "pack_move_comfort_activation": "Frais d'activation de l'option Confort",
+  "pack_move_comfort_activation": "Les frais d'activation de l'option Confort sont de <span class=\"text-price\">{{ price }}</span> HT",
+  "pack_move_resume_install_fees": "Les frais d'installation / création de ligne sont de <span class=\"text-price\">{{ price }}</span> HT",
```

We have to avoid to use HTML inside content files.
You can complete it like this:

```suggestion
  "pack_move_resume_install_fees": "Les frais d'installation / création de ligne sont de {{price}} HT",
```

then in your HTML (sample):
```
<p data-translate="" data-translate-values="{  price: '<span class=\"text-price\">{{ price }}</span>' }"></p>
```

---

ok done